ui/core/src/users/Form.js

```diff
@@ -0,0 +1,209 @@
+  }
+
+  constructor(props) {
+    super(props)
+
+    this.state = {
+      user: Object.assign({}, props.user),
+      hasError: false,
+      isLoading: false,
+      errors: null
+    }
+  }
+
+  changeUser(attrs) {
+    const user = Object.assign({}, this.state.user, attrs);
+    this.setState(Object.assign({}, this.state, {user}));
+  }
+
+  handleSubmit(event) {
+    event.preventDefault();
+    const user = this.state.user;
+
+    this.props.relay.commitUpdate(
+      new AddUserMutation({
+        user
+      }),{
+        onSuccess: response => {
+          console.log(response);
+          this.setState({isLoading: false});
+          // TODO: get rid of reload and use router.
+          // TODO: Redirect to the updated list of users without reloading page and show errors in the form
+          // Hint: Tabs js is interferring with React in members/index
+          // window.location.reload();
+        },
+        onFailure: transaction => {
+          console.log(transaction.getError().source);
+          this.setState({
+            hasError: true,
+            isLoading: false
+          });
+        }
+      }
+    );
+
+    this.setState({isLoading: true});
+  }
+
+  render() {
+    const { title } = this.props;
+    const { isLoading, user, hasError } = this.state;
+    console.log(this.state.user);
+    return (
+      <main>
+        <Grid>
+          <Cell col={12}>
+            <h1 styleName="form-title">{title}</h1>
+          </Cell>
+        </Grid>
+        <Grid>
+          <Cell col={12}>
+            { this.state.hasError && (
+              <span styleName="login-error">
+
+              </span>
+            )}
+          </Cell>
+        </Grid>
+        <form id="user-form" onSubmit={e => this.handleSubmit(e)}>
+          {isLoading && (<div styleName="loading">
+            <Spinner />
+          </div>)}
+          <Grid>
+            <Cell offset={2} col={4}>
+              <Textfield
+                id="first_name"
+                labelText="Nombre(s)"
+                onChange={e => this.changeUser({
+                  first_name: e.target.value
+                })}
+              />
+            </Cell>
+            <Cell col={4}>
+              <Textfield
+                id="last_name"
+                labelText="Apellido(s)"
+                onChange={e => this.changeUser({
+                  last_name: e.target.value
+                })}
+              />
+            </Cell>
+          </Grid>
+          <Grid>
+            <Cell offset={2} col={4}>
+              <Textfield
+                id="email"
+                labelText="Email"
+                onChange={e => this.changeUser({
+                  email: e.target.value
+                })}
+              />
+            </Cell>
+            <Cell col={2}>
+              <Select instanceId='role'
+                name='role'
+                styleName="dropdown"
+                value={user.role}
+                options={[
```

Comment:
Ideally, we should provide values like this from a graphql. Please add a TODO so we don't forget to fix/improve this in the future